### PR TITLE
Updated readme.md to align with style guide

### PR DESCRIPTION
**Pull request by celanthe**
Made first-pass changes to bring the contributor guide in line with the Kubernetes documentation style guide.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: contributors/guide/README.md [hunk 1]
```diff
@@ -66,21 +68,22 @@ Consequently, it is wholly dependent on its community to provide a productive, f
...
-The [Community Membership Document](/community-membership.md) covers membership processes and roles.
+Kubernetes is a large, lively, friendly open-source community.
+As many open source projects often do, it depends on new people becoming members and regular code contributors. 
+The [Community Membership Document](/community-membership.md) covers membership processes and roles. 
+Please consider joining Kubernetes, and if you wish to, making your way up the contributor ladder!
 
 ## Communication
 
-- [General Information](/communication)
+- [General Information](/communication) relating to Kubernetes communicaton policies
```
Comment (markyjackson-taulia) [inline contributors/guide/README.md:86]: `communicaton` is misspelled causing the check to fail

**Review by markyjackson-taulia — CHANGES_REQUESTED** [review]

File: contributors/guide/README.md [hunk 2]
```diff
@@ -5,100 +5,174 @@ aliases: [ "/guide" ]
...
+- [Contributor Workshops](#kubernetes-contributor-workshops)
 - [Community](#community)
   - [Communication](#communication-1)
   - [Events](#events)
     - [Meetups](#meetups)
   - [Mentorship](#mentorship)
 - [Advanced Topics](#advanced-topics)
 
-# Prerequisites
+## Prerequisites
```
Comment (mrbobbytables) [inline contributors/guide/README.md:69]: The items beneath here were intended to be sub headings. The could probably goto h3 `###` but I'd also update the ToC at the top to align.

File: contributors/guide/README.md [hunk 3]
```diff
@@ -5,100 +5,174 @@ aliases: [ "/guide" ]
...
 
-Before submitting code to the project you should first take care of the following prerequisites.
-These steps are checked by a bot during your first submission, so doing these steps first will make your first contribution easier: 
+Before submitting code to Kubernetes, you should first complete the following 
+prerequisites.
+These steps are checked automatically by [a bot]
+during your first submission. 
+Completing these steps will make your first contribution easier:
```
Comment (mrbobbytables) [inline contributors/guide/README.md:75]:
```suggestion
These steps are checked automatically by [a bot] during your first submission.
Completing these steps will make your first contribution easier: 
```
These can be consolidated to match the 80 character guideline. 

There are other instances like this in the doc, I'd consolidate where needed.

File: contributors/guide/README.md [hunk 4]
```diff
@@ -5,100 +5,174 @@ aliases: [ "/guide" ]
...
+- [General Information] relating to Kubernetes communication policies
 
 ## Events
 
-Kubernetes participates in KubeCon + CloudNativeCon, held three times per year in China, Europe and in North America.
-Information about these and other community events is available on the CNCF [events](https://www.cncf.io/events/) pages.
+Kubernetes participates in KubeCon + CloudNativeCon, held three times per year in 
+China, Europe and in North America.
+Information about these and other community events is available on the CNCF 
+[Events]pages.
```
Comment (mrbobbytables) [inline contributors/guide/README.md:138]:
```suggestion
[Events] pages.
```
missing ` `.

File: contributors/guide/README.md [hunk 5]
```diff
@@ -5,100 +5,174 @@ aliases: [ "/guide" ]
...
-We depend on new people becoming members and regular code contributors, so we would like you to come join us!
-The [Community Membership Document](/community-membership.md) covers membership processes and roles.
+Kubernetes is a large, lively, friendly open-source community.
+As many open source projects often do, 
+it depends on new people becoming members and regular code contributors. 
+The [Community Membership Document] 
+covers membership processes and roles. 
+Please consider joining Kubernetes, and making your way up the contributor ladder!
 
 ## Communication
```
Comment (mrbobbytables) [inline contributors/guide/README.md:129]:
```suggestion
### Communication
```
The items here through Mentoring I think are intended to be subheadings under community.

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks! - I've included mostly style suggestions for a first pass :)

File: contributors/guide/README.md [hunk 6]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
+## Welcome
 
 Have you ever wanted to contribute to the coolest cloud technology?
-We will help you understand the organization of the Kubernetes project and direct you to the best places to get started.
-You'll be able to pick up issues, write code to fix them, and get your work reviewed and merged.
+This guide will help you understand the overall organization 
+of the Kubernetes project, 
+and direct you to the best places to get started contributing. 
+You'll be able to pick up issues, write code to fix them, 
+and get your work reviewed and merged.
```
Comment (mrbobbytables) [inline contributors/guide/README.md:39]:
```suggestion
This guide will help you understand the overall organization  of the Kubernetes
project, and direct you to the best places to get started contributing.  You'll
be able to pick up issues, write code to fix them, and get your work reviewed
and merged.
```

File: contributors/guide/README.md [hunk 7]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
 
-This document is the single source of truth for how to contribute to the code base.
-Feel free to browse the [open issues](https://github.com/kubernetes/community/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fcontributor-guide) and file new ones, all feedback welcome!
+This document is the single source of truth for how to 
+contribute to the code base. Feel free to browse the  [open issues] 
+and file new ones, all feedback is welcome!
```
Comment (mrbobbytables) [inline contributors/guide/README.md:43]:
```suggestion
This document is the single source of truth for how to  contribute to the code
base. Feel free to browse the  [open issues]  and file new ones, all feedback
is welcome!
```

File: contributors/guide/README.md [hunk 8]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
-## Sign the CLA
+Before you can contribute to Kubernetes, you will need to sign the
+[Contributor License Agreement].  
 
-Before you can contribute, you will need to sign the [Contributor License Agreement](/CLA.md).  
+### Code of Conduct
 
-## Code of Conduct
+Please make sure to read and observe the 
+[Code of Conduct] and [Community Values]
```
Comment (mrbobbytables) [inline contributors/guide/README.md:74]:
```suggestion
Please make sure to read and observe the [Code of Conduct] and
[Community Values].
```

File: contributors/guide/README.md [hunk 9]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
 
-Please make sure to read and observe our [Code of Conduct](/code-of-conduct.md) and [Community Values](/values.md)
+### Setting up your development environment
 
-## Setting up your development environment
+It is not required to set up a developer environment in order to 
+contribute to Kubernetes.
```
Comment (mrbobbytables) [inline contributors/guide/README.md:79]:
```suggestion
It is not required to set up a developer environment in order to  contribute to
Kubernetes.
```

File: contributors/guide/README.md [hunk 10]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
 
-If you haven’t set up your environment, check the [developer resources](/contributors/devel/README.md#setting-up-your-dev-environment-coding-and-debugging).
+If you plan to contribute code changes, 
+review the [developer resources]
+page for how to set up your environment.
```
Comment (mrbobbytables) [inline contributors/guide/README.md:83]:
```suggestion
If you plan to contribute code changes,  review the [developer resources] page
for how to set up your environment.
```

File: contributors/guide/README.md [hunk 11]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
+Consequently, it is wholly dependent on its community to provide a productive, 
+friendly and collaborative environment.
 
-- Read and review the [Community Expectations](expectations.md) for an understanding of code and review expectations.
-- See [Community Membership](/community-membership.md) for a list the various responsibilities of contributor roles. You are encouraged to move up this contributor ladder as you gain experience.
+- Read and review the [Community Expectations] for an 
+understanding of code and review expectations.
+- See [Community Membership] for a list the various 
+responsibilities of contributor roles. 
+- You are encouraged to move up this contributor ladder as you gain experience.
```
Comment (mrbobbytables) [inline contributors/guide/README.md:95]:
```suggestion
- Read and review the [Community Expectations] for an  understanding of code
  and review expectations.
- See [Community Membership] for a list the various  responsibilities of
  contributor roles. 
- You are encouraged to move up this contributor ladder as you gain experience.
```

File: contributors/guide/README.md [hunk 12]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
 
-# Kubernetes Contributor Playground
+## Kubernetes Contributor Playground
 
-If you are looking for a safe place, where you can familiarize yourself with (some of) the Kubernetes Project's review and pull request processes, then the [Kubernetes Contributor Playground](https://github.com/kubernetes-sigs/contributor-playground/) is the right place for you.
+If you are looking for a safe place, where you can familiarize yourself with 
+the pull request and issue review process 
+in Kubernetes, then the [Kubernetes Contributor Playground]
+is the right place for you.
```
Comment (mrbobbytables) [inline contributors/guide/README.md:102]:
```suggestion
If you are looking for a safe place, where you can familiarize yourself with 
the pull request and issue review process  in Kubernetes, then the
[Kubernetes Contributor Playground] is the right place for you.
```

File: contributors/guide/README.md [hunk 13]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
 
-## Contributor Workshops
+### Contributor Workshops
 
-A [Youtube playlist](https://www.youtube.com/playlist?list=PL69nYSiGNLP3M5X7stuD7N4r3uP2PZQUx) of the New Contributor workshop has been posted, and an outline of the video content can be found [here](/events/2018/05-contributor-summit). 
+A [Youtube playlist]
+of the New Contributor workshop has been posted.
+An outline of the video content can be found [here].
```
Comment (mrbobbytables) [inline contributors/guide/README.md:108]:
```suggestion
A [Youtube playlist] of the New Contributor workshop has been posted. An outline
of the video content can be found [here]. 
```

File: contributors/guide/README.md [hunk 14]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
-If you haven't noticed by now, we have a large, lively, and friendly open-source community.
-We depend on new people becoming members and regular code contributors, so we would like you to come join us!
-The [Community Membership Document](/community-membership.md) covers membership processes and roles.
+Kubernetes is a large, lively, friendly open-source community.
+As many open source projects often do, 
+it depends on new people becoming members and regular code contributors. 
+The [Community Membership Document] 
+covers membership processes and roles. 
+Please consider joining Kubernetes, 
+and making your way up the contributor ladder!
```
Comment (mrbobbytables) [inline contributors/guide/README.md:118]:
```suggestion
Kubernetes is a large, lively, friendly open-source community. As many open
source projects often do,  it depends on new people becoming members and regular
code contributors.  The [Community Membership Document] covers membership
processes and roles. Please consider joining Kubernetes,  and making your way
up the contributor ladder!
```

File: contributors/guide/README.md [hunk 15]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
 
-## Events
+### Events
 
-Kubernetes participates in KubeCon + CloudNativeCon, held three times per year in China, Europe and in North America.
-Information about these and other community events is available on the CNCF [events](https://www.cncf.io/events/) pages.
+Kubernetes participates in KubeCon + CloudNativeCon, 
+held three times per year in China, Europe and in North America.
+Information about these and other community events is available on the CNCF 
+[Events] pages.
```
Comment (mrbobbytables) [inline contributors/guide/README.md:129]:
```suggestion
Kubernetes participates in KubeCon + CloudNativeCon, held three times per year
in China, Europe and in North America. Information about these and other
community events is available on the CNCF [Events] pages.
```

File: contributors/guide/README.md [hunk 16]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
-
-# Advanced Topics
-
-This section includes things that need to be documented, but typical contributors do not need to interact with regularly.
-
-- [OWNERS files](owners.md) - The Kubernetes organizations are managed with OWNERS files, which outline which parts of the code are owned by what groups.
+All Kubernetes meetups follow the general 
+[Cloud Native Computing Foundation Guidelines]
+You may also contact CNCF Staff driving the Community Groups 
+(previously known as Meetups) program by email (meetups@cncf.io)
```
Comment (mrbobbytables) [inline contributors/guide/README.md:136]:
```suggestion
All Kubernetes meetups follow the general [Cloud Native Computing Foundation Guidelines]
You may also contact CNCF Staff driving the Community Groups  (previously known
as Meetups) program by email (meetups@cncf.io)
```
The first line does exceed 80, but it's not too much more and it's a long link.

File: contributors/guide/README.md [hunk 17]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
+
+### Mentorship
+
+Learn more about the Kubernetes [mentoring initiatives].
+Join past and present Kubernetes Contributors 
+for a live question-and-answer session during [Meet Our Contributors].
```
Comment (mrbobbytables) [inline contributors/guide/README.md:142]:
```suggestion
Learn more about the Kubernetes [mentoring initiatives]. Join past and present
Kubernetes Contributors  for a live question-and-answer session during
[Meet Our Contributors]. 
```

File: contributors/guide/README.md [hunk 18]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
+Join past and present Kubernetes Contributors 
+for a live question-and-answer session during [Meet Our Contributors]. 
+
+## Advanced Topics
+
+This section includes things that need to be documented, 
+but typical contributors do not need to interact with regularly.
+
+- [OWNERS files] - The Kubernetes organizations are managed with OWNERS files, 
+which outline which parts of the code are owned by what groups.
```
Comment (mrbobbytables) [inline contributors/guide/README.md:150]:
```suggestion
This section includes things that need to be documented, but typical contributors
do not need to interact with regularly.

- [OWNERS files] - The Kubernetes organizations are managed with OWNERS files,
  which outline which parts of the code are owned by what groups.
```

File: contributors/guide/README.md [hunk 19]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
 description:
   A small list of things that you should read and be familiar with before you
   get started with contributing. This includes such things as signing the
-  Contributor License Agreement, familiarizing yourself with our Code of Conduct
-  and more.
+  Contributor License Agreement, 
+  familiarizing yourself with our Code of Conduct, and more.
```
Comment (mrbobbytables) [inline contributors/guide/README.md:9]:
```suggestion
  Contributor License Agreement, familiarizing yourself with our Code of Conduct,
  and more.
```

File: contributors/guide/README.md [hunk 20]
```diff
@@ -5,100 +5,165 @@ aliases: [ "/guide" ]
...
 
 - [Welcome](#welcome) - this page 
-- [Prerequisites](#prerequisites) - things you need to complete before contributing
-- [Your First Contribution](first-contribution.md) - things you need to know before your first contribution
-- [Contributing](contributing.md) - the main guide with contributor information
+- [Prerequisites](#prerequisites) - tasks you need to complete before 
+  you can start contributing to Kubernetes
+- [Your First Contribution](first-contribution.md) - things you'll need to know 
+  before making your first contribution
+- [Contributing](contributing.md) - the main reference guide to contributing
```
Comment (mrbobbytables) [inline contributors/guide/README.md:55]: I didn't catch these the other day, but these could both be ref links

**Review by mrbobbytables — COMMENTED** [review]
I made some suggestions around targeting 80 characters for the line length. 👍

File: contributors/guide/README.md [hunk 21]
```diff
@@ -5,100 +5,174 @@ aliases: [ "/guide" ]
...
+- [Setting Up Your Developer Environment](#setting-up)
+- [Community Expectations and Roles](#community-expectations)
+- [Contributor Playground](#contributor-playground)
+- [Contributor Workshops](#contributor-workshops)
+- [Community](#community)
+- [Communication](#communication)
+- [Events](#events)
+- [Meetups](#meetups)
+- [Mentorship](#mentorship)
+- [Advanced Topics](#advanced-topics)
```
[the remark this answers is not in the document]
Comment (celanthe) [reply contributors/guide/README.md]: Done!

File: contributors/guide/README.md (hunk 19 above)
Comment (celanthe) [reply contributors/guide/README.md]: Done!

File: contributors/guide/README.md (hunk 20 above)
Comment (celanthe) [reply contributors/guide/README.md]: Done!

File: contributors/guide/README.md (hunk 18 above)
Comment (celanthe) [reply contributors/guide/README.md]: Done!

File: contributors/guide/README.md [hunk 22]
```diff
@@ -5,100 +5,174 @@ aliases: [ "/guide" ]
...
 Welcome to Kubernetes! This guide is broken up into the following sections. 
-It is recommended that you follow these steps in order. 
+It is recommended that you follow these steps in order: 
 
 - [Welcome](#welcome) - this page 
-- [Prerequisites](#prerequisites) - things you need to complete before contributing
-- [Your First Contribution](first-contribution.md) - things you need to know before your first contribution
-- [Contributing](contributing.md) - the main guide with contributor information
+- [Prerequisites](#prerequisites) - tasks you need to complete before you can start 
+contributing to Kubernetes
```
[the remark this answers is not in the document]
Comment (celanthe) [reply contributors/guide/README.md]: Done!

File: contributors/guide/README.md (hunk 3 above)
Comment (celanthe) [reply contributors/guide/README.md:63]: Done!

**Comment by mrbobbytables** [comment]
/approve
Thanks!

**Pull request merged**